Update docs about indexer unit file (#1551)
manticoresoftware/manticoresearch: manual/Data_creation_and_modification/Adding_data_from_external_storages/Plain_tables_creation.md

* 1: there was a problem while indexing (and if `--rotate` was specified, it was skipped) or an operation emitted a warning
* 2: indexing went OK, but the `--rotate` attempt failed

> Also, you can run the indexer via a systemctl unit file:

Reviewer: ```suggestion
Also, you can run `indexer` via a systemctl unit file:
```

```shell
systemctl start --no-block manticore-indexer
```

> Or, in case you want to index a specific table:

Reviewer: ```suggestion
Or, in case you want to build a specific table:
```

```shell
systemctl start --no-block manticore-indexer@desired-table-name
```

> 

Reviewer: ```suggestion
Find more information about scheduling `indexer` via systemd below.

```

`ignore_non_plain` allows you to completely ignore warnings about skipping non-plain tables. The default is 0 (not ignoring).
<!-- proofread -->

### Schedule indexer

You have two approaches to schedule indexer running. First way - its crontab. You need to install cron and set commands.
The second way its Using a Timer with a User-Defined Schedule:

To create the timer unit files for your system, you should place them in the appropriate directory where systemd looks for such unit files. On most Linux distributions, this directory is typically /etc/systemd/system. Here's how to add the paths to the unit files for both the user-defined and predefined schedules:

#### Predefined Schedule:

1. Create a timer unit file for your custom schedule, as mentioned in the previous response.
```shell
cat << EOF > /etc/systemd/system/manticore-indexer@.timer
[Unit]
Description=Run ManticoreSearch's indexer by schedule

[Timer]
OnCalendar=minutely
RandomizedDelaySec=5m

> Unit=manticore-indexer@%i.timer

Reviewer 2: It should be a service (`Unit=manticore-indexer@%i.service`)

> ### Schedule indexer
> 
> You have two approaches to schedule indexer running. First way - its crontab. You need to install cron and set commands.
> The second way its Using a Timer with a User-Defined Schedule:
> 
> 
> To create the timer unit files for your system, you should place them in the appropriate directory where systemd looks for such unit files. On most Linux distributions, this directory is typically /etc/systemd/system. Here's how to add the paths to the unit files for both the user-defined and predefined schedules:
> 
> #### Predefined Schedule:
> 
> 1. Create a timer unit file for your custom schedule, as mentioned in the previous response.
> ```shell
> cat << EOF > /etc/systemd/system/manticore-indexer@.timer
> [Unit]
> Description=Run ManticoreSearch's indexer by schedule
> 
> [Timer]
> OnCalendar=minutely
> RandomizedDelaySec=5m
> Unit=manticore-indexer@%i.timer
> 
> [Install]
> WantedBy=timers.target
> EOF
> ```
> 2. Edit the timer unit for your specific needs.
> 3. Enable the timer:
> ```shell
> systemctl enable manticore-indexer@idx1.timer
> ```
> 4. Start the timer:
> ```shell
> systemctl start manticore-indexer@idx1.timer
> ```
> 5. Repeat steps 2-4 for any additional timers.

Reviewer: ```suggestion
### Schedule indexer via systemd

You have two approaches to scheduling indexer runs. The first way is the classical method of using crontab. The second way is using a systemd timer with a user-defined schedule. To create the timer unit files, you should place them in the appropriate directory where systemd looks for such unit files. On most Linux distributions, this directory is typically `/etc/systemd/system`. Here's how to do it:

1. Create a timer unit file for your custom schedule:
   ```shell
   cat << EOF > /etc/systemd/system/manticore-indexer@.timer
   [Unit]
   Description=Run ManticoreSearch's indexer on schedule
   [Timer]
   OnCalendar=minutely
   RandomizedDelaySec=5m
   Unit=manticore-indexer@%i.service
   [Install]
   WantedBy=timers.target
   EOF
   ```
   More on the `OnCalendar` syntax and examples can be found here <https://www.freedesktop.org/software/systemd/man/latest/systemd.time.html#Calendar%20Events>.

2. Edit the timer unit for your specific needs.
3. Enable the timer:
   ```shell
   systemctl enable manticore-indexer@idx1.timer
   ```
4. Start the timer:
   ```shell
   systemctl start manticore-indexer@idx1.timer
   ```
5. Repeat steps 2-4 for any additional timers.
```
